src/LayerPlan.cpp

```diff
@@ -382,6 +382,11 @@ GCodePath& LayerPlan::addTravel(Point p, bool force_comb_retract)
         bypass_combing = true; // first travel move is bogus; it is added after this and the previous layer have been planned in LayerPlanBuffer::addConnectingTravelMove
         first_travel_destination = p;
         first_travel_destination_is_inside = is_inside;
+        if (layer_nr == 0 && extruder->settings.get<bool>("retraction_hop_enabled"))
```

Comment:
I think this needs to check both `retraction_enable` and `retraction_hop_enabled`.

Reply:
Adressed in bd4d9ab